README.md

```diff
@@ -146,6 +146,8 @@ Benchmark a package over a set of revisions.
 - `--path <arg>`: Path of the package.
 - `--bench-on <arg>`: If the script is not set, this specifies the revision at which
   to download `benchmark/benchmarks.jl` from the package.
+- `--nsamples-load-time <arg>`: Number of samples to take when measuring load time of
+    the package (default: 1). (This means starting a Julia process for each sample.)
```

Comment:
just double checking, there are a few places where it defaults to 5 below. Is this docstring outdated?

Reply:
Yes, good catch!